scripts/zones/La_Theine_Plateau/npcs/Vicorpasse.lua

```diff
@@ -27,7 +27,7 @@ function onTrigger(player,npc)
 	if(player:getCurrentMission(SANDORIA) == THE_RESCUE_DRILL) then
 		local MissionStatus = player:getVar("MissionStatus");
 		
-		if(MissionStatus == 4) then
+		if(MissionStatus >= 0 and MissionStatus <= 4) then

```

Comment:
No, I've def missed him at the beginning, prior to the bronze sword. And every single source online specifically states that he has to be spoken with first, immediately after getting the mission from the gate guard. You can see I have reason to doubt this is wrong, being as they all say it and my own memory says it, thought it may have changed and wiki(s) never got the memo. That would be a strange thing to get changed after over 10 years of FFXI though.